documentation/source/commands/canu.rst

```diff
@@ -13,37 +13,37 @@ canu
                errorRate=0.X \
               [other-options] \
               [-pacbio-raw | -pacbio-corrected | -nanopore-raw | -nanopore-corrected] *fastq
-  
+
     By default, all three stages (correct, trim, assemble) are computed.
     To compute only a single stage, use:
       -correct  - generate corrected reads
       -trim     - generate trimmed reads
       -assemble - generate an assembly
-  
+
     The assembly is computed in the (created) -d <assembly-directory>, with most
     files named using the -p <assembly-prefix>.
-  
+
     The genome size is your best guess of the genome size of what is being assembled.
     It is used mostly to compute coverage in reads.  Fractional values are allowed: '4.7m'
     is the same as '4700k' and '4700000'
-  
+
     The errorRate is not used correctly (we're working on it).  Don't set it
     If you want to change the defaults, use the various utg*ErrorRate options.
-  
+
     A full list of options can be printed with '-options'.  All options
-    can be supplied in an optional sepc file.
-  
+    can be supplied in an optional spec file.
```

Comment:
If you could please verify this should be "spec" as I wasn't entirely sure, thanks.